Projects/UOContent/Items/Weapons/Abilities/ForceArrow.cs

```diff
@@ -0,0 +1,161 @@
+using Server.Spells;
+using System;
+using System.Collections.Generic;
+
+namespace Server.Items
+{
+    public class ForceArrow : WeaponAbility
+    {
+        public override int BaseMana => 20;
+
+        public override bool RequiresTactics(Mobile from) => false;
+
+        public override void OnHit(Mobile attacker, Mobile defender, int damage)
+        {
+            if (!Validate(attacker) || !CheckMana(attacker, true))
+                return;
+
+            ClearCurrentAbility(attacker);
+
+            attacker.SendLocalizedMessage(1074381); // You fire an arrow of pure force.
+            defender.SendLocalizedMessage(1074382); // You are struck by a force arrow!
+
+            if (.4 > Utility.RandomDouble())
+            {
+                defender.Combatant = null;
+                defender.Warmode = false;
+            }
+
+            ForceArrowInfo info = GetInfo(attacker, defender);
+
+            if(info == null)
+            {
+                BeginForceArrow(attacker, defender);
+            }
+            else if (info.Timer != null && info.Timer.Running)
+            {
+                info.Timer.IncreaseExpiration();
+
+                BuffInfo.RemoveBuff(defender, BuffIcon.ForceArrow);
+                BuffInfo.AddBuff(defender, new BuffInfo(BuffIcon.ForceArrow, 1151285, 1151286, info.DefenseChanceMalus.ToString()));
+            }
+
+            if (defender.Spell is Spell spell && spell.IsCasting)
```

Comment:
Missing braces for this if statement. Our style guide requires braces for all conditionals to make the code more readable and require less changes if we need to modify it later.

Reply:
Yes, i like it with braces too.